modules/administration-guide/partials/con_a-workspace-example.adoc

```diff
@@ -7,7 +7,30 @@ This section describes a {prod-short} workspace example.
 
 The following devfile defines the {prod-short} workspace:
 
-include::example$snip_{project-context}-a-workspace-example.adoc[]
+[source,yaml]
+----
+schemaVersion: 2.1.0
+metadata:
+  name: bash
+components:
+  - name: tools
+    container:
+      image: quay.io/devfile/universal-developer-image:ubi8-0e189d9
```

Comment:
Is there any mechanism for keeping this image tag up-to-date? The tag used here is already 8 months old.

I wonder if it makes more sense for the sample in the docs to use `ubi8-latest` to avoid confusion if tools are added to the UBI image for example.

```suggestion
      image: quay.io/devfile/universal-developer-image:ubi8-latest
```

Reply:
You are perfectly right.